content/k8s/manual-app-deployment/postgresql-installation/_index.md

```diff
@@ -5,7 +5,8 @@ tags:
 - kubernetes
 - postgresql
 prerequisites:
-  hard: []
+  hard: 
+  - k8s/manual-app-deployment/nginx-installation
```

Comment:
They actually don't need to have Nginx before PostgreSQL, they can do this in a different order.

Maybe flag as as soft prereq? Or remove the prereqs?